# 目标信息

std_msgs/Header header

# 目标类别名称
int32 target_category

# 目标位置，相机坐标系下：右方x为正，下方y为正，前方z为正
float32[2] position